tests/Service/CollapsedBroadcastsService/AbstractCollapsedBroadcastServiceTest.php

```diff
@@ -36,4 +43,18 @@ protected function service(): CollapsedBroadcastsService
             $this->mockServiceRepository
         );
     }
+
+    /**
+     * @param Category[] $categories
+     * @return int[]
+     */
+    protected function extractDbAncestryIds(array $categories): array
```

Comment:
This is only used in one test class, is it worth putting it in the abstract test case?

Reply:
yeah, you are right